### PR TITLE
Add inventory docs button link

### PR DESCRIPTION
**Pull request by laviro**
Selection_322

### CODE REVIEW

File: webpack/ForemanInventoryUpload/Components/PageHeader/PageHeader.js [hunk 1]
```diff
@@ -2,22 +2,26 @@ import React from 'react';
...
 
 const PageHeader = () => (
   <React.Fragment>
     <Grid.Row>
-      <Grid.Col xs={12}>
+      <Grid.Col xs={6}>
         <h1 className="inventory_title">{INVENTORY_PAGE_TITLE}</h1>
       </Grid.Col>
+      <Grid.Col xs={6}>
+        <AutoUploadSwitcher />
```
Comment (laviro) [inline webpack/ForemanInventoryUpload/Components/PageHeader/PageHeader.js:16]: moved upload switcher to the row above as it feels less dense with all of the other buttons

File: webpack/ForemanInventoryUpload/Components/PageHeader/components/DocsButton/DocsButton.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Button, Icon } from 'patternfly-react';
+import {
+  INVENTORY_DOCS_LINK,
```
Comment (ShimShtein) [inline webpack/ForemanInventoryUpload/Components/PageHeader/components/DocsButton/DocsButton.js:4]: Can we consume the URL through helper?
I want a helper that will use documentation controller in Foreman for latest core version. In previous versions, we can still use direct link.

Comment (laviro) [reply webpack/ForemanInventoryUpload/Components/PageHeader/components/DocsButton/DocsButton.js]: That's a great idea, changed to consume it from an helper

**Comment by laviro** [comment]
rebased, ready for another round :)

File: webpack/ForemanInventoryUpload/ForemanInventoryHelpers.js [hunk 3]
```diff
@@ -2,3 +2,6 @@ import { foremanUrl } from '../ForemanRhCloudHelpers';
 
 export const inventoryUrl = path =>
   foremanUrl(`/foreman_inventory_upload/${path}`);
+
+export const getInventoryDocsUrl = () =>
+  'https://access.redhat.com/products/subscription-central';
```
Comment (ShimShtein) [inline webpack/ForemanInventoryUpload/ForemanInventoryHelpers.js:7]: Shouldn't we use foreman's documentation function:

``` js
foremanUrl + '/links/manual?root_url=https://access.redhat.com/products/&section=subscription-central'
```

Comment (laviro) [reply webpack/ForemanInventoryUpload/ForemanInventoryHelpers.js:7]: not sure why, it is leading to https://theforeman.org/manuals/2.0/quickstart_guide.html

Comment (laviro) [reply webpack/ForemanInventoryUpload/ForemanInventoryHelpers.js]: Using this worked:
```js
foremanUrl(
    `/links/manual/+?root_url=${URI.encode(
      'https://access.redhat.com/products/subscription-central'
    )}`
  );
```

**Comment by ShimShtein** [comment]
Merged, thanks @laviro!

**Pull request merged**